synapse/lib/stormlib/aha.py

```diff
@@ -85,6 +111,73 @@ async def _methAhaGet(self, svcname, filters=None):
         proxy = await self.runt.snap.core.reqAhaProxy()
         return await proxy.getAhaSvc(svcname, filters=filters)
 
+    async def _methCallPeerApi(self, svcname, apiname, timeout=None, skiprun=None):
+        '''
+        Call an API on an AHA service.
+
+        Args:
+            svcname (str): The name of the AHA service to call
+            apiname (str, tuple): The API name or todo tuple (name, args, kwargs)
+            timeout (int): Optional timeout in seconds
+            skiprun (bool): Optional flag to skip run checks
+        '''
+        svcname = await s_stormtypes.tostr(svcname)
+
+        if isinstance(apiname, str):
+            apiname = (apiname, (), {})
+        else:
+            apiname = await s_stormtypes.toprim(apiname)
+            if not isinstance(apiname, (list, tuple)):
+                raise s_exc.BadArg(mesg='API name must be a string or tuple')
+
+            name = apiname[0]
```

Comment:
We can simply require them to provide a valid todo tuple rather than needing to add so much extra logic here.

Reply:
I couldn't find any todo prim norm calls and because this is passed from storm we need a good way to transform and enforce elements to StormTypes. Completely open to better ways on improving this, to note: I added a bunch of tests around it too.

Reply:
Mostly considering when we inevitably need a `todo` tuple in a different API :) Perhaps something like `$lib.todo()` (or maybe in a util somewhere?) which uses uses the fact that we can add args/kwargs methods to storm from python and just returns the result of `synapse.common.todo()`?

Reply:
I created a new `$lib.todo` let me know if its OK or needs some additional work.